UndertaleModLib/Models/UndertaleGlobalInit.cs

```diff
@@ -29,7 +31,7 @@ public void Serialize(UndertaleWriter writer)
     public void Unserialize(UndertaleReader reader)
     {
         _code = new UndertaleResourceById<UndertaleCode, UndertaleChunkCODE>();
-        _code.Unserialize(reader); // TODO: reader.ReadUndertaleObject if one object starts with another one
+       _code.Unserialize(reader);   // Cannot use ReadUndertaleObject, as that messes things up.
```

Comment:
OK so this should be good to merge now, but I think you accidentally removed a space at the start of the line.